site/en/docs/privacy-sandbox/attribution-reporting/register-attribution-source/index.md

```diff
@@ -0,0 +1,310 @@
+```
+
+Review the [example code](https://github.com/GoogleChromeLabs/trust-safety-demo/blob/8f3d874b79ab0c8a15822fbcd09e94042aee7dcd/conversion-measurement/functions/views/adtech/ad-click.pug) for more information.
+
+
+#### Using a script
+
+Call `window.open()` with `attributionsrc`:
+
+```javascript
+window.open(
+  "https://shoes.example/...",
+  "_blank",
+  "attributionsrc");
+```
+
+To be taken into account, this method must be called within 5 seconds of user interaction.
+
+<p>
+Instead of adding <code>attributionsrc</code> by itself, for an image or script you can specify a single URL value:
+
+
+<pre>
+&lt;a href=... attributionsrc="https://a.example/register-source">
+</pre>
+
+In the JavaScript case, if you give `attributionsrc` a value, make sure
+to encode that URL in case it contains special characters such as '=' that
+would cause the parameter to be improperly parsed. 
+        
+Encode as follows:
+<p>
+
+```javascript
+const encodedUrl = encodeURIComponent(
+  'https://adtech.example/attribution_source?ad_id=...');
+window.open(
+  "https://shoes.example/landing",
+   "_blank",
+   attributionsrc=${encodedUrl});
+```
+
+`attributionsrc` can also take a space-separated list of URLs as illustrated here with an anchor tag:
+
+```html
+<!-- With an anchor tag -->
+<a href=... attributionsrc="https://a.example/register-source 
+  https://b.example/register-source"> 
+```
+
+or as it is here using `window.open()`.
+
+```javascript
+// With window.open()
+window.open('...', '_blank', attributionsrc=${encodedUrl1}
+  attributionsrc=${encodedUrl2})
+```
+
+In such cases, both URLs receive `navigation-source-eligible` `attributionsrc`
+requests (requests that include the `Attribution-Reporting-Eligible` header).
+
+#### `attributionsrc` with or without a value
+
+As you saw above, you can specify `attributionsrc` without a URL. You can also specify a
+single URL. Additionally, for sources only (not applicable to [triggers](/docs/privacy-sandbox/attribution-reporting/register-attribution-trigger/)), you can use a space-separated list of URLs.
+
+Using URLs causes the browser to initiate a **separate** keepalive fetch request—one
+for each URL—which includes the `Attribution-Reporting-Eligible` request
+header.
+
+This is useful if you want to make the source registration by responding to a
+request that is separate from the element's main request.  
+
+For example, if you need to register sources for clicks on an anchor element,
+you may not actually be in control of the destination; in this case, you'll want
+a configuration whereby you send the source registration header as a response to a request
+that is separate from the navigation, and that you can completely control. By
+specifying an explicit value for `attributionsrc`, you're instructing the
+browser to make that extra request and configuring its destination.
+
+Click the tab above for <a href="#tabs">Views step 1</a>
+
+  </web-tab>
+  <web-tab title="Views step 1">
+<p>
+
```

Comment:
We should punctuate this sentence on both tabs, however, my issue with this instruction is it's a bit confusing. We're redirecting them to the tab with the link so what we need to say is something like "For instructions on how to register..., go to [the tab](# link back to the tabs) and select **Views step 1**."

Reply:
good idea thanks!